### PR TITLE
odf-cli: add config file for odf-cli repo

### PR DESCRIPTION
**Pull request by sp98**
This PR adds the yaml files to setup prow CLI for odf-cli repo

### CODE REVIEW

File: core-services/prow/02_config/red-hat-storage/odf-cli/OWNERS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- BlaineEXE
+- nb-ohad
+options: {}
+reviewers:
+- travisn
+- subhamkrai
+- agarwal-mudit
+- BlaineEXE
+- nb-ohad
+- parth-gr
```
Comment (parth-gr) [inline core-services/prow/02_config/red-hat-storage/odf-cli/OWNERS:20]: needs a extra line

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Pull request merged**